core/src/main/java/io/aiven/klaw/config/SecurityConfigNoSSO.java

```diff
@@ -160,46 +115,13 @@ public AuthenticationProvider activeDirectoryLdapAuthenticationProvider() {
     return provider;
   }
 
-  private void ldapAuthentication(AuthenticationManagerBuilder auth) throws Exception {
-    try {
-      log.info("Ldap authentication configured.");
-      if (!checkLdapConnectivity()) {
-        throw new KlawException("Cannot connect to Ldap !!");
-      }
-
-      if (encryptionType != null && encryptionType.equals("bcrypt")) {
-        auth.ldapAuthentication()
-            .userSearchBase(ldapBase)
-            .userDnPatterns(userDnPatterns)
-            .contextSource()
-            .url(ldapUrls)
-            .managerDn(managerUserName)
-            .managerPassword(managerPwd)
-            .and()
-            .passwordCompare()
-            .passwordEncoder(new BCryptPasswordEncoder())
-            .passwordAttribute(passwordAttribute);
-      }
-
-    } catch (Exception e) {
-      log.error("Cannot connect to Ldap !! ", e);
-      shutdownApp();
-      throw new Exception("Cannot connect to Ldap !!");
-    }
-  }
-
-  private void dbAuthentication(AuthenticationManagerBuilder auth) throws Exception {
-    log.info("Db authentication configured.");
-    auth.userDetailsService(inMemoryUserDetailsManager());
-  }
-
   @Bean
   public InMemoryUserDetailsManager inMemoryUserDetailsManager() throws Exception {
     final Properties globalUsers = new Properties();
     if (authenticationType != null && authenticationType.equals(DATABASE.value)) {
+      log.info("Db authentication configured.");
       log.info("Loading all users !!");
```

Comment:
I believe the second log statement should be debug.

Reply:
Agree, done.